packages/network/test/unit/agent_spec.ts

```diff
@@ -414,6 +414,36 @@ describe('lib/agent', function () {
         })
       })
     })
+
+    context('HttpAgent - Upstream target URL', function () {
+      beforeEach(function () {
```

Comment:
this could be nested in the previous `HttpAgent` context instead of being a sibling, `this.agent` and `this.request` don't need to be duplicated here

Reply:
Fixed: https://github.com/cypress-io/cypress/commit/b9bed2d1b7b91d3a30e8d3ad6693c5b3198f496f